modules/affile/affile-dest.c

```diff
@@ -97,7 +97,7 @@ struct _AFFileDestWriter
   time_t last_msg_stamp;
   time_t last_open_stamp;
   time_t time_reopen;
-  struct iv_timer reap_timer;
+  struct iv_timer retry_timer;
```

Comment:
I don't think, that additional 64 bytes (size of an iv_timer) in the struct does matter something. Because one timer should be registered at one time, so the ivykis won't have problem with this. So I don't understand why it is a waste.